modules/carousel/jetpack-carousel.js

```diff
@@ -225,12 +225,15 @@ jQuery(document).ready(function($) {
 
 			container = $('<div></div>')
 				.addClass('jp-carousel-wrap')
-				.addClass( 'jp-carousel-transitions' );
-
+				.addClass( 'jp-carousel-transitions' )
```

Comment:
Travis failure due to jshint picking up the missing semicolon at this line. Could you add it back?